src/components/settings/PermanentFilterSettings.tsx

```diff
@@ -20,7 +20,8 @@ export const PermanentFilterSettings = () => {
 
   const fetchPolicy = graphqlFetchPolicy({ isConnected, isMainserverUp, refreshTime });
 
-  const { state: excludedDataProviderIds, dispatch } = usePermanentFilter();
+  const { excludeDataProviderIds: excludedDataProviderIds, dataProviderDispatch: dispatch } =
```

Comment:
`excludeDataProviderIds` should be `dataProviderState` or am I wrong?

Reply:
I answered here. I think that'll be enough.

https://github.com/smart-village-solutions/smart-village-app-app/pull/1063#discussion_r1548134400

Reply:
wouldn't it be more readable to refactor the code to `const { excludeDataProviderIds, dataProviderDispatch }` instead of renaming things. so the code needs to be adjusted to not use `excludedDataProviderIds` and `dispatch` anymore. no one knows what `dispatch` is related to.